torch/fx/proxy.py

```diff
@@ -441,8 +442,10 @@ def find_tracer(a):
             return tracer.create_proxy('call_method', orig_method.__name__, args, kwargs)
         else:
             if isinstance(orig_method, torch._ops.PyOperator):
-                # TODO: Define how to symbolically trace PyOperators
-                raise RuntimeError("Unable to symbolically trace PyOperators")
+                try:
+                    return orig_method.create_proxy(tracer, args, kwargs)
+                except Exception as e:
+                    raise RuntimeError(f"Unable to symbolically trace PyOperator {orig_method.name()}")
```

Comment:
Why aren't we just propagating the error as is here

Reply:
Yeah, we should do that.